src/libsyntax/attr/builtin.rs

```diff
@@ -252,7 +252,7 @@ fn find_stability_generic<'a, I>(sess: &ParseSess,
                 }
             }
 
-            match meta.ident_str().expect("not a stability level") {
```

Comment:
Did you mean to remove `expect` call here?

Reply:
Yes, the impossible case will now return an empty string and hit the `unreachable!()` arm in the `match`.